content/concepts/tokens/common-misconceptions-about-freezes.ja.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: freezes.html
+blurb: XRP Ledgerのフリーズ機能について、よくある誤解を解いていきます。
+labels:
+  - トークン
+---
+# トークンの凍結に関するよくある誤解
+
+PayPalのような中央集権的なサービスがアカウントを停止して資金にアクセスできないようにするのと同様に、Ripple社などがXRPを凍結することができるというのはよくある誤解です。XRP Ledgerには[凍結機能](freezes.html)がありますが、これは発行トークンにのみ使用可能で、XRPには使用できません。 **XRPを凍結することは誰にもできません**。
+
+XRP Ledgerのトークンは、[XRPとは根本的に異なる](currency-formats.html#comparison)ものです。トークンは常にトラストライン上に存在し、それは凍結される可能性があります。XRPはアカウントに含まれており、凍結されることはありません。
+
+## XRPは単なるRipple社のトークンではないのか？ <!-- STYLE_OVERRIDE: just -->
```

Comment:
Like in the other PR, the STYLE_OVERRIDE instructions are irrelevant for Japanese and can be safely removed

```suggestion
## XRPは単なるRipple社のトークンではないのか？
```